doc/build-unix.md

```diff
@@ -98,11 +98,11 @@ See the section "Disable-wallet mode" to build Bitcoin Core without wallet.
 
 Optional:
 
-    sudo apt-get install libminiupnpc-dev (see --with-miniupnpc and --enable-upnp-default)
+    sudo apt-get install libminiupnpc-dev # (see --with-miniupnpc and --enable-upnp-default)
```

Comment:
Alternatively this (...) could go after Optional in the previous line. The less to copy/paste the better I'd say, and comments don't really make that much sense here.
(same below)